planemo/training/tutorial.py

```diff
@@ -183,7 +183,7 @@
 > ### {{ '{%' }} icon hands_on {{ '%}' }} Hands-on: Data upload
 >
 > 1. Create a new history for this tutorial
-> 2. Import the files from [Zenodo]({{ zenodo_link }}) or from the shared data library
+> 2. Import the files from [Zenodo]({{ zenodo_link }}) or from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`):
```

Comment:
The failed tests are relevant. Since this is a template string used to generate a template string, I think this should be:
```suggestion
> 2. Import the files from [Zenodo]({{ '{{' }} page.zenodo_link {{ '}}' }}) or from the shared data library (`GTN - Material` -> `{{ '{{' }} page.topic_name {{ '}}' }}` -> `{{ '{{' }} page.title {{ '}}' }}`):
```

Reply:
thanks @nsoranzo!